docs/deployment/advanced-configuration.rst

```diff
@@ -136,6 +137,67 @@ There are several default values that are attached to each created metadata. If
         https://purl.org/fair-metrics/FM_F1A: https://www.ietf.org/rfc/rfc3986.txt
         https://purl.org/fair-metrics/FM_A1.1: https://www.wikidata.org/wiki/Q8777
 
+FDP Index
+=========
+
+You can turn your FAIR Data Point instance into a FDP Index that can be contacted by other FDPs and harvests metadata from them.
+
+Hosting FDP Index
+-----------------
+
+To enable FDP Index mode on your FDP server, just simply adjust your ``application.yml`` file:
+
+.. code:: yaml
+    
+    # application.yml
+
+    fdp-index:
+        enabled:  true
+
+
+Then for the FDP client, you need to use ``fairdata/fairdatapoint-index-client`` Docker image for browsing indexed FDPs and searching harvested metadata. In case you want to use your deployment both as FDP and FDP Index, you can deploy both FDP and FDP Index client applications. The configuration of both clients is identical.
+
+.. code:: yaml
+
+    # docker-compose.yml
+
+    version: '3'
+    services:
+
+    # ...
+
+    index_client:
+      image: fairdata/fairdatapoint-index-client:1.9.0
+      restart: always
+      # ...
+
+
+Connecting to FDP Index
+-----------------------
+
+By default, FDPs use https://home.fairdatapoint.org as their primary FDP Index that they ping every 7 days. You can adjust that in  your ``application.yml`` file if needed:
+
+.. code:: yaml
+    
+    # application.yml
+
+    ping:
+        endpoint: https://my-index.example.com
+        interval: 86400000 # milliseconds
+
+You can also set multiple endpoints if needed:
+
+.. code:: yaml
+    
+    # application.yml
+
+    ping:
+        endpoint: >
```

Comment:
Having `>` here correct? I am checking in case it is a mistake :-)

Reply:
Yes, it is multiline string in yaml: https://symfony.com/doc/current/components/yaml/yaml_format.html#strings

This and separated by comma were the simplest possibilities. YAML list would require to have whole configuration described using classes. That might be done in the future.